pages/index.js

```diff
@@ -1,9 +1,16 @@
 import Form from '../src/components/Form/Form';
 
+import dynamic from 'next/dynamic';
+
 export default function Home() {
+	const Card = dynamic(() => import('../src/components/Card/Card'), {
+		ssr: false,
+	});
+
```

Comment:
Please add a little comment for that, so everyone seeing this in future, knows why you did that.

Reply:
All right